src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -107,6 +107,20 @@ limitations under the License.
             <include cluster="Power Source" client="false" server="true" clientLocked="true" serverLocked="true"></include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-electricalsensor</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Electrical Sensor</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0510</deviceId>
+        <class>Utility</class>
+        <scope>Node</scope>
+        <clusters lockOthers="true">
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true"></include>
+            <include cluster="Electrical Energy Measurement" client="false" server="true" clientLocked="false" serverLocked="false"></include>
+            <include cluster="Electrical Power Measurement" client="false" server="true" clientLocked="false" serverLocked="false"></include>
```

Comment:
Shouldn't clientLocked be true for both of these?

Reply:
Fixed.